crates/sui-indexer-alt-schema/migrations/2024-12-02-185822_add_coin_balance_owner_kind/metadata.toml

```diff
@@ -0,0 +1 @@
+run_in_transaction = false
```

Comment:
what does this do?

Reply:
It makes it so that the migration is not run in a transaction with other pending transactions, it's run independently. It's required because the `CREATE INDEX CONCURRENTLY` statements need to be run outside of a transaction.